### DISCUSSION TITLE
View Keyframe on Web

### DISCUSSION THREAD

**Discussion by TRINHHOANGANH** [stella-cv/stella_vslam#473] [General]
Dear @All
I run the Stella_vslam project, it produces 1 directory including: keyframe and transforms.json. I want to see a list of keyframes on the web viewer.
Has anyone tried some libraries like panolens, marzipano, krpano, three.js to view keyframes on web?
Please give me your solution.

**Comment by Skwangles** [comment]
Extremely old question, but I'll answer for anyone looking. 
The project has been updated to include a 'socket_viewer' and 'socket_publisher', which uses THREE.js to view
